ui/src/app/domains/views/DomainsList/DomainsTable/DomainsTable.tsx

```diff
@@ -2,8 +2,7 @@ import { MainTable } from "@canonical/react-components";
 import { useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 
-import domainURLs from "../../../urls";
-
+import domainURLs from "app/domains/urls";
```

Comment:
@bartaz If it's more than one level up or down it's probably safer to use absolute paths instead of relative paths